sklearn/linear_model/tests/test_ridge.py

```diff
@@ -487,6 +487,35 @@ def test_class_weights():
     assert_array_almost_equal(clf.intercept_, clfa.intercept_)
 
 
+def test_class_weight_vs_sample_weight():
```

Comment:
Upsetting that we're duplicating code and then going to need to rein it in.

On 10 June 2015 at 14:49, Trevor Stephens <notifications@github.com> wrote:

> In sklearn/linear_model/tests/test_ridge.py
> <https://github.com/scikit-learn/scikit-learn/pull/4838#discussion_r32087957>
> :
>
> > @@ -487,6 +487,35 @@ def test_class_weights():
> >      assert_array_almost_equal(clf.intercept_, clfa.intercept_)
> >
> >
> > +def test_class_weight_vs_sample_weight():
>
> Upsetting how? That it's propagating outside of common tests?
>
> Best I can trace right now was @dsullivan7 <https://github.com/dsullivan7>
> 's #3931 <https://github.com/scikit-learn/scikit-learn/pull/3931>
> (somewhat similar test on multiplicative weights), then RF & D-Tree in
> #3961 <https://github.com/scikit-learn/scikit-learn/pull/3961> .. Now
> here in Ridge. Cannot recall now the exact lineage I used from the RF/tree
> PR tests. Don't see any duplicated inline comments outside of RF/Tree/Ridge
> on git FWIW.
>
> Note that in tree-based classifier's I tested for equivalency in
> feature_importances_, and here it's coef_.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/scikit-learn/scikit-learn/pull/4838/files#r32087957>.
>